python_scripts/ensemble_hyperparameters.py

```diff
@@ -121,3 +127,11 @@
 # Here, we tune the `n_estimators` but be aware that using early-stopping as
 # in the previous exercise will be better.
 # ```
+
+# %% [markdown]
+# Now we estimate the generalization performance of the best model
+# using the test set.
+
+# %%
+score = -grid_search.score(data_test, target_test)
+print(f"On average, our GBDT regressor makes an error of {score:.2f} k$")
```

Comment:
Here again it would be nice to add a comment to quickly compare this value to the CV values observed during parameter tuning (they should match approximately but it can be even slightly better because the final model has been retrained on more that the inner CV models of the grid search procedure).

Reply:
Thanks, done!

Reply:
```suggestion
print(f"On average, our GBDT regressor makes an error of {error:.2f} k$")
```

Reply:
```suggestion
print(f"On average, our GBDT regressor makes an error of {error:.2f} k$")
```